Python/Product/PythonTools/PythonTools/Repl/InteractiveWindowProvider.cs

```diff
@@ -107,14 +124,17 @@ class InteractiveWindowProvider {
             return null;
         }
 
-        public IVsInteractiveWindow OpenOrCreate(string replId) {
+        public IVsInteractiveWindow OpenOrCreate(string replId) => OpenOrCreate(replId, null);
+
+        public IVsInteractiveWindow OpenOrCreate(string replId, Func<IInteractiveEvaluator, bool> predicate) {
```

Comment:
Looks like override with `predicate != null` isn't used.

Reply:
Not yet, no. But the API should mirror the `Open()` function.

Reply:
`OpenOrCreate` looks for specific `replId` that can be skipped by predicate and as a result, we will get several windows with identical id's. Is that expected?

Reply:
Yes. We can have multiple windows with the same REPL id - that's why we need the predicate to tell them apart without interrupting the search (which is the underlying cause of this bug)